plutus-core-interpreter/src/PlutusCoreInterpreter/CekMachine.hs

```diff
@@ -0,0 +1,96 @@
+module PlutusCoreInterpreter.CekMachine where
+
+import           Language.PlutusCore
+import           Language.PlutusCore.Constant
+import           Language.PlutusCore.Evaluation.MachineException (MachineError (..), MachineException (..))
+import           Language.PlutusCore.Evaluation.Result           (EvaluationResult (..))
+import           Language.PlutusCore.View
+import           PlutusPrelude
+
+import           Data.IntMap                                     (IntMap)
+import qualified Data.IntMap                                     as IntMap
+
+type Plain f = f TyName Name ()
```

Comment:
I'm not sure about what to do here. I do not want to pollute the global namespace with this non-descriptive name. Do you have any name suggestions?
We can turn it into a type family and make it work over `Type` too :)

Reply:
Or we could be very boring and have two type synonyms ;)
```
type PlainTerm = Term TyName Name ()
type PlainType = Type TyName ()
```

Reply:
Also

```haskell
type PlainValue = Value TyName Name ()
type PlainProgram = Program TyName Name ()
```

I'm fine with this.

Reply:
But that changes a lot across the code base, so belongs to another PR.

Reply:
I don't think we should enforce using them , we can just put them there if you want them. Many places can't use them because they're generic over the annotation.